src/Raven.Server/Documents/ETL/Providers/Raven/RavenEtl.cs

```diff
@@ -76,6 +76,13 @@ protected override void LoadInternal(IEnumerable<ICommandData> items, JsonOperat
             }
         }
 
+        protected override bool ShouldFilterOutSystemDocument(bool isHiLo)
+        {
+            // if we transfer all documents to the same collections (no script specified) then don't exclude HiLo docs
+
```

Comment:
We need to document entire ETL feature. I included that info here: http://issues.hibernatingrhinos.com/issue/RDoc-665